### PR TITLE
VIH-8099 added moj module and applied style for header component

### PR DESCRIPTION
**Pull request by omio-ma**
### JIRA link ###
https://tools.hmcts.net/jira/browse/VIH-8099

### Change description ###
- added moj node module
- updated header component to use moj styles

**Does this PR introduce a breaking change?** 

```
[  ] Yes
[ x ] No
```

### CODE REVIEW

File: ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/shared/header/header.component.html [hunk 1]
```diff
@@ -1,29 +1,33 @@
...
+
+        <image src="/assets/images/moj-logotype-crest.png" xlink:href="" class="moj-header__logotype-crest-fallback-image" width="40" height="40"></image>
+
+      </svg>
+
+      <a class="moj-header__link moj-header__link--organisation-name" [routerLink]="">HM Courts &amp; Tribunals Service</a>
+
     </div>
+    <div class="moj-header__content">
```
Comment (kevin-hmcts) [inline ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/shared/header/header.component.html:17]: The __  is this standard recall a previous comment suggesting it should be - are you bale to confirm?

Comment (omio-ma) [reply ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/shared/header/header.component.html:12]: yea, thats the style based on the documentation:
https://design-patterns.service.justice.gov.uk/components/header/

**Review by simonwkelso — APPROVED** [review]
Have got a walkthrough and demo from muki - looks good and appears to function as it should. As for Kevins comment, the '__' is default class name in the component.

**Review by kevin-hmcts — DISMISSED** [review]
LGTM

**Review by FarronF — APPROVED** [review]

**Review by simonwkelso — APPROVED** [review]
LGTM

**Pull request merged**